app/src/main/java/com/timchenko/playlistmaker/ui/media/PlaylistDetailsFragment.kt

```diff
@@ -121,12 +147,8 @@ class PlaylistDetailsFragment: Fragment() {
             .transform(CenterCrop())
             .into(binding.playlistDetailsCoverMenu)
 
-        binding.playlistDetailsCover.contentDescription = playlist.name + " : " + playlist.description
-
-        BottomSheetBehavior.from(binding.bottomSheetPlaylistTracks).apply {
-            peekHeight = binding.root.height - binding.buttonPlaylistShare.bottom - resources.getDimensionPixelSize(R.dimen.value_24)
-            state = BottomSheetBehavior.STATE_COLLAPSED
-        }
+        binding.playlistDetailsCover.contentDescription =
+            playlist.name + " : " + playlist.description
```

Comment:
🍏 Можно воспользоваться Kotlin templates, чтобы не создавать промежуточные строки:
```suggestion
            "$playlist.name : $playlist.description"
```

https://www.baeldung.com/kotlin/string-templates

Reply:
исправлено